engines/order_management/spec/services/order_management/order/updater_spec.rb

```diff
@@ -32,6 +32,7 @@ module Order
           allow(order).to receive_message_chain(:all_adjustments, :tax, :additional, :sum).and_return(20)
           allow(order).to receive_message_chain(:all_adjustments, :enterprise_fee, :sum).and_return(10)
           allow(order).to receive_message_chain(:all_adjustments, :shipping, :sum).and_return(5)
+          allow(order).to receive_message_chain(:shipment_adjustments, :tax, :inclusive, :sum).and_return(5)
```

Comment:
I agree with @mkllnk 's earlier comment that all this feels like we're kind of implementing half of it with stubs and it feels like a symptom of an OOP design that doesn't fit our needs. We'll need to look into it in the future, once we refactor a bit more.